install_with_docker.sh

```diff
@@ -66,6 +66,21 @@ else
     echo 'Not Supported Architecture'
 fi
 
+fetch_ip_info() {
+    response=$(curl -s https://ipinfo.io)
+
+    ip=$(echo "$response" | awk -F'"' '/ip/{print $4}')
```

Comment:
When I run these in my terminal, and then `echo $ip` I get two lines; the first is my IP and the second is `https://ipinfo.io/missingauth`. I think we just want the first line. For all the other extracted parameters I got the expected results